server/py/services/api/api/endpoints/user_secrets.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from http import HTTPStatus
+
+import fastapi
+from fastapi.concurrency import run_in_threadpool
+from sqlalchemy.orm import Session
+
+import mlrun.common.schemas
+
+import framework.api.deps
+import framework.utils.auth.verifier
+import services.api.crud
+
+router = fastapi.APIRouter(prefix="/user-secrets/tokens")
```

Comment:
The prefix here is valid for all endpoint handlers in this file.
if we will support other user secrets that are not tokens, this will not be true.
I suggest having `prefix="/user-secrets"` here, and in the handler add `tokens`